en/homework/advanced/otp-concurrency/otp-concurrency-solution.exs

```diff
@@ -0,0 +1,47 @@
+defmodule SimpleQueue do
```

Comment:
I think this homework assignment is a great start for our beginners. What do you think of also including (and updating if need-be) this old homework assignment on the GenServer topic in which we prompt students to build a "bank" with a GenServer? https://github.com/elixirschool/homework/tree/master/simple_bank

I think it might be a nice "stretch" homework assignment for anyone looking for a challenge.